cdbot/cogs/fun.py

```diff
@@ -99,7 +101,12 @@ def __init__(self, bot: Bot):
         if message.channel.id == QUOTES_CHANNEL_ID and (
             message.author.id == QUOTES_BOT_ID or message.mentions is not None
         ):
-            conn = await asyncpg.connect()
+            conn = await asyncpg.connect(host=base64.b64decode(environ.get("PGHOST")),
+                                         port=base64.b64decode(environ.get("PGPORT")),
+                                         user=base64.b64decode(environ.get("PGUSER")),
+                                         password=base64.b64decode(environ.get("PGPASSWORD")),
+                                         database=base64.b64decode(environ.get("PGDATABASE")))
```

Comment:
This is not nice, could this be stored in the constants file so you can pass it straight into the function?